demo/README.md

```diff
@@ -0,0 +1,140 @@
+# Standalone Rook
```

Comment:
i thought the discussion was that this content would be mostly in the main readme, based on the rook-client container. we have to show the object, file, and block usage on the main readme